src/arrays/exercise005/Exercise005.java

```diff
@@ -0,0 +1,36 @@
+package arrays.exercise005;
+
+import java.util.Arrays;
+
+/***
+ * Write a Java program to find the index of an array element.
+ */
+public class Exercise005 {
+    public static void main(String[] args) {
+        int[] intArr = {1, 3, 5, 6, 8, 12};
+
+        int testVal = 1;
+        int indexValue = findIndexValue(intArr, testVal);
+        System.out.println(indexValue);
+    }
+
+    public static int findIndexValue(int[] arr, int val) {
+        boolean present = Arrays.stream(arr)
```

Comment:
Ta część jest niepotrzebna. Druga część wystarczy. Tu nie ma cudów, stream zrobi najpierw pętlę po wszystkich elementach i potem jeszcze raz to zrobisz sam. Raz wystarczy.